metadata-ingestion/schedule_docs/airflow.md

```diff
@@ -0,0 +1,11 @@
+# Using Airflow
+
+If you are using Apache Airflow for your scheduling then you might want to use it for scheduling your ingestion receipes. To do that you can follow these steps
+- Create a receipe file e.g. `recipe.yml`
+- Ensure [DataHub CLI](../../docs/cli.md) is installed in your airflow environment
+- Create a sample DAG file like [`generic_recipe_sample_dag.py`](../src/datahub_provider/example_dags/generic_recipe_sample_dag.py). This will read your DataHub ingestion recipe file and run it.
+- Deploy the DAG file into airflow for scheduling.
```

Comment:
Is `deploy` the right verb here? maybe add another sentence that: Typically this involves checking in the DAG file into your dags repo for Airflow?